src/org/zaproxy/zap/extension/brk/BreakAPI.java

```diff
@@ -91,6 +118,60 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
 		} else if (ACTION_BREAK_ON_ID.equals(name)) {
 			extension.setBreakOnId(params.getString(PARAM_KEY), params.getString(PARAM_STATE).toLowerCase().equals("on"));
 
+		} else if (ACTION_CONTINUE.equals(name)) {
+			extension.getBreakpointManagementInterface().cont();
+
```

Comment:
Drop the extra lines here? (Down to 130) Maybe this is a style thing....to me it seems disjointed.